src/scores/output.py

```diff
@@ -0,0 +1,106 @@
+"""Creating the final Excel output of the project."""
+
+from datetime import date
+
+from pandas import DataFrame, ExcelWriter
+from xlsxwriter import Workbook
+from xlsxwriter.format import Format
+from xlsxwriter.worksheet import Worksheet
+
+from src.config import tables
```

Comment:
Naming - on reflection, I think we should rename this. "tables" sounds like it should contain the tables, not be a path to a directory holding table data. Maybe `tables_dir`?